UI/BuildDesignatorWnd.cpp

```diff
@@ -1051,7 +1051,8 @@ void BuildDesignatorWnd::CenterOnBuild(int queue_idx, bool open) {
             MapWnd* map = ClientUI::GetClientUI()->GetMapWnd();
             map->CenterOnObject(system_id);
             if (open) {
-                SelectSystem(system_id);
+                HumanClientApp::GetApp()->GetClientUI().GetMapWnd()->SelectSystem(system_id);
+                //SelectSystem(system_id);
```

Comment:
> Wow, that is quite a lecture for what I already said was accidental-- a simple "Yes, I was referring to the commented line" seems it would have been 100% sufficient. 

Yes this kind of lecture is unfortunately required, because things like that are getting committed all the time.  If it is because of carelessness, ignorance or malicious intent doesn't matter because the outcome, hard to understand code and programmers intent, is the same.

> Are you trying to say you deem it to be bad practice to comment out code while working on replacing it?  That the code must be fully deleted before the replacement code is ready and working?

Yes.  Try out commiting onto a local branch, rebasing and cleaning up before publishing instead.

> and I would not expect the extra commit to make it into master -- I would expect this to PR to be accepted with a squash commit. Are you saying I should have immediately done a force push with a clean single commit replacing the existing one?

Yes.  Someone will fuck it up in the future and do the wrong thing (like you already have seen in you last comment).  Having rules/processes in place is not a thing to bother people, but to avoid errors that can be avoided by following a process.